post-type/module-base.php

```diff
@@ -851,15 +851,15 @@ public static function dt_user_list_filters( $filters, $post_type ){
                 }
                 $filters["tabs"][] = [
                     "key" => "all",
-                    "label" => _x( "All", 'List Filters', 'disciple-tools-training' ),
+                    "label" => _x( "Default Filters", 'List Filters', 'disciple-tools-training' ),
                     "count" => $total_all,
                     "order" => 10
                 ];
                 // add assigned to me filters
                 $filters["filters"][] = [
                     'ID' => 'all',
                     'tab' => 'all',
-                    'name' => _x( "All", 'List Filters', 'disciple-tools-training' ),
+                    'name' => _x( "All Trainings", 'List Filters', 'disciple-tools-training' ),
```

Comment:
@mikeallbutt let's copy the All %s format

Reply:
Ok sounds good. Do I need to do something different in the code then?

Reply:
yes. Copy the structure from https://github.com/DiscipleTools/disciple-tools-theme/pull/1609/commits/aee4d6c78bf176401670c55225447d0e96bc960c